libpretixsync/src/main/java/eu/pretix/libpretixsync/db/QuestionLike.java

```diff
@@ -45,9 +45,9 @@ public String getDefault() {
 
     public List<String> getDependencyValues() { return new ArrayList<>(); }
 
-    public String clean_answer(String answer, List<QuestionOption> opts) throws AbstractQuestion.ValidationException {
+    public String clean_answer(String answer, List<QuestionOption> opts, boolean requiredOverride) throws ValidationException {
```

Comment:
From the naming of the parameter, it's not clear to me what true/false *mean*. Maybe `ìgnoreRequired` or `allOptional`  would be clearer? (I believe we use `all_optional` in a similar scenario in the server-side code)